docs/quickstart/install-and-use-a-package-using-the-dotnet-cli.md

```diff
@@ -103,7 +103,7 @@ Congratulations on installing and using your first NuGet package!
 
 ## Related video
 
-> [!Video https://docs.microsoft.com/shows/NuGet-101/Install-and-Use-a-NuGet-Package-with-the-NET-CLI-3-of-5/player]
+> [!Video https://Microsoft technical documentation/shows/NuGet-101/Install-and-Use-a-NuGet-Package-with-the-NET-CLI-3-of-5/player]
```

Comment:
This is a false positive in the automated tool, which breaks the embedded video

Reply:
Fixed, thanks for the feedback!